src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -357,6 +359,11 @@ def load_arguments(self, _):
     with self.argument_context('network application-gateway rule', min_api='2017-06-01') as c:
         c.argument('redirect_config', help='The name or ID of the redirect configuration to use with the created rule.')
 
+    with self.argument_context('network application-gateway identity', min_api='2019-04-01') as c:
+        c.argument('identity_type', help="The type of identity used for the resource.", arg_type=get_enum_type(['SystemAssigned', 'UserAssigned', 'None']))
```

Comment:
If users want to assign system identities, how can they assign role can scope?

Reply:
Currently, the user cannot assign system identities. The back-end server will refuse such request.